gui/static/js/interface.js

```diff
@@ -31,79 +32,71 @@ function getPower(){
 	return $("#power").val();
 }
 
-function getBotID() {
-    return $("#botlist").val();
-}
-
-function getScript() {
-    return $("#data").val();
-}
-
-function sendMotors(fl, fr, bl, br) {
-	$.ajax({
-		method: "POST",
-		url: "/commandBot",
-		data: JSON.stringify({
-			name: getBotID(),
-			fl: fl,
-			fr: fr,
-			bl: bl,
-			br: br
-		}),
-		processData: false,
-		contentType: 'application/json'
-	});
-}
-
-function startLogging() {
-	$.ajax({
-		method: "POST",
-		url: "/logdata",
-		data: JSON.stringify({
-			name: getBotID()
-		}),
-		processData: false,
-		contentType: 'application/json'
-	});
-}
-
-function sendScript() {
-    $.ajax({
-    		method: "POST",
-    		url: "/sendScript",
-    		data: JSON.stringify({
-    			name: getBotID(),
-    			script: getScript()
-    		}),
-    		processData: false,
-    		contentType: 'application/json'
-    	});
-}
-
-$("#send").click(function(event) {
-    sendScript();
-});
-
-<<<<<<< HEAD
-
-
-
-
-
-
-=======
-$('#xbox-off').click(function() {
-	// ajax post to backend to remove a bot from list.
-	$.ajax({
-		method: "POST",
-		url: '/stopXbox',
-		dataType: 'json',
-		contentType: 'application/json',
-		success: function properlyRemoved(data) {
-		    console.log("TODO");
-		}
-	});
-});
+// function getBotID() {
+//     return $("#botlist").val();
+// }
+//
+// function getScript() {
+//     return $("#data").val();
+// }
+//
+// function sendMotors(fl, fr, bl, br) {
+// 	$.ajax({
+// 		method: "POST",
+// 		url: "/commandBot",
+// 		data: JSON.stringify({
+// 			name: getBotID(),
+// 			fl: fl,
+// 			fr: fr,
+// 			bl: bl,
+// 			br: br
+// 		}),
+// 		processData: false,
+// 		contentType: 'application/json'
+// 	});
+// }
+//
+// function startLogging() {
+// 	$.ajax({
+// 		method: "POST",
+// 		url: "/logdata",
+// 		data: JSON.stringify({
+// 			name: getBotID()
+// 		}),
+// 		processData: false,
+// 		contentType: 'application/json'
+// 	});
+// }
+//
+// function sendScript() {
+//     $.ajax({
+//     		method: "POST",
+//     		url: "/sendScript",
+//     		data: JSON.stringify({
+//     			name: getBotID(),
+//     			script: getScript()
+//     		}),
+//     		processData: false,
+//     		contentType: 'application/json'
+//     	});
+// }
+//
+// $("#send").click(function(event) {
+//     sendScript();
+// });
+//
+// $('#xbox-off').click(function() {
+// 	// ajax post to backend to remove a bot from list.
+// 	$.ajax({
+// 		method: "POST",
+// 		url: '/stopXbox',
+// 		dataType: 'json',
+// 		contentType: 'application/json',
+// 		success: function properlyRemoved(data) {
+// 		    console.log("TODO");
+// 		}
+// 	});
+// });
 
 // when adding a bot
 $('#addBot').click(function() {
```

Comment:
This still works?

Reply:
Wait this isn't run after #26, right?

Reply:
not sure. these files aren't imported or used by the new GUI at all, actually. they're just being kept around for reference/as a list of features we haven't converted.

Reply:
you can just remove then, then.